servers/rendering/renderer_scene_cull.cpp

```diff
@@ -2659,7 +2659,7 @@ bool RendererSceneCull::_light_instance_update_shadow(Instance *p_instance, cons
 	return animated_material_found;
 }
 
-void RendererSceneCull::render_camera(const Ref<RenderSceneBuffers> &p_render_buffers, RID p_camera, RID p_scenario, RID p_viewport, Size2 p_viewport_size, uint32_t p_jitter_phase_count, float p_screen_mesh_lod_threshold, RID p_shadow_atlas, Ref<XRInterface> &p_xr_interface, RenderInfo *r_render_info) {
+void RendererSceneCull::render_camera(const Ref<RenderSceneBuffers> &p_render_buffers, RID p_camera, RID p_scenario, RID p_viewport, Size2 p_viewport_size, uint32_t p_jitter_phase_count, float p_screen_mesh_lod_threshold, RID p_shadow_atlas, Ref<RefCounted> &p_xr_interface, RenderInfo *r_render_info) {
```

Comment:
This might not be needed, see:
```
#ifdef _3D_DISABLED
// RendererSceneCull::render_camera is empty when 3D is disabled, but
// it and RenderingMethod::render_camera have a parameter for XRInterface.
#define XRInterface RefCounted
#else // 3D enabled
#include "servers/xr/xr_interface.h"
#endif // _3D_DISABLED
```

You likely just need to replace `_3D_DISABLED` by `XR_DISABLED` here.